wagtail/users/tests/test_admin_views.py

```diff
@@ -1567,6 +1567,12 @@ def test_hide_publish_permissions(self):
         # Should not show inputs for publish permissions on models without DraftStateMixin
         self.assertNotInHTML("Can publish advert", html)
 
+    def test_view_permission_if_model_has_verbose(self):
```

Comment:
I think we need to be a bit more explicit about what's being tested here. 

Maybe add an expect statement that the model's verbose name is somewhere in the right column also. 

It's not clear here that we are using a model in the permissions listing that is using a verbose name. 

We are only checking that nothing has a word after 'can view'. 

Also. Probably add a docstring here with a bit of explanation and maybe an issue link. 

This might seem a bit nitpicky but this kind of bug will be super easy to miss and we want it to break the tests if we change something accidentally.